portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/ext/EnvironmentVariables.testcase

```diff
@@ -0,0 +1,45 @@
+@component-name = "portal-util-misc"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Ext";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Verify portal-ext properties can be set with environment variables after setting company-id-properties=true via jvm argument."
+	@priority = "3"
+	test ConfigurePortalExtPropertyViaJVM {
+		property test.environment.variables = "LIFERAY_PROPS_BY_COMPANY_20097=feature.flag.COMMERCE-8087=true${line.separator}CATALINA_OPTS=\"$CATALINA_OPTS -Dcompany-id-properties=true\"";
+		property test.name.skip.portal.instance = "EnvironmentVariables#ConfigurePortalExtPropertyViaJVM";
+
+		ImportExport.openImportExportAdmin();
+
+		ImportExport.gotoImport();
```

Comment:
We need a control check here in case this feature flag gets turned on, we will see it fail and know to change the feature flag, kind of like how you had a control check with the clamd anti-virus check.

It will also be more clean to move calling of the environment variables outside of the base server startup and just directly calling it from poshi since other tests will most likely never use this feature.

So basically -> Check FeatureFlag disabled -> Shutdown -> Enable Feature flag via Environment Variable ->Startup -> Check FeatureFlag enabled